volumes/miovision/dev_notebooks/intersection_tmc_notebook03test.py

```diff
@@ -0,0 +1,388 @@
+"""Used to test notebook 03. Identical to intersection_tmc.py on 2020-10-01
```

Comment:
Should update the `dev_notebooks/README` to include an explanation of this script and the 03 notebook

Reply:
Done